docs/user-guide/workers.md

```diff
@@ -176,48 +175,52 @@ For highlight goodness and simplicity sake, we are going to use an `mpy` script
 ```
 
 **main.py**
-```Python title="MicroPython bootstrapping a Pyodide worker"
+```Python title="MicroPython's main.py: bootstrapping a Pyodide worker."
 from pyscript import PyWorker, document
 
-# bootstrap the pyodide worker with optional config too
-# the worker here is:
-#   * owned by this script, no JS or Pyodide code in the same page can access it
-#   * it allows pre-sync methods exposure
-#   * it exposes a ready Promise to await pyodide on the worker side
-#   * it then allows using post-sync (utilities exposed by pyodide)
+# Bootstrap the Pyodide worker, with optional config too.
+# The worker is:
+#   * Owned by this script, no JS or Pyodide code in the same page can access
+#     it.
+#   * It allows pre-sync methods to be exposed.
+#   * It has a ready Promise to await for when Pyodide is ready in the worker. 
+#   * It allows the use of post-sync (methods exposed by Pyodide in the
+#     worker).
 worker = PyWorker("worker.py", type="pyodide")
 
-# expose an utility that can be invoke *out of the box* in worker.py
+# Expose a utility that can be immediately invoked in the worker. 
 worker.sync.greetings = lambda: print("Pyodide bootstrapped")
 
 print("before ready")
-# await for Pyodide to complete its bootstrap
+# Await until Pyodide has completed its bootstrap, and is ready.
 await worker.ready
 print("after ready")
 
-# await any exposed utility exposed via Pyodide
+# Await any exposed methods exposed via Pyodide in the worker.
 result = await worker.sync.heavy_computation()
 print(result)
 
-# show the result at the end of the body
+# Show the result at the end of the body.
 document.body.append(result)
 
-# here we free memory and get rid of everything
+# Free memory and get rid of everything in the worker.
 worker.terminate()
 ```
 
 **worker.py**
-```Python title="A Pyodide worker"
+```Python title="The worker.py script runs in the Pyodide worker."
 from pyscript import sync
 
-# use any already exposed utility from main.py
+# Use any methods from main.py on the main thread.
 sync.greetings()
 
-# expose any method meant to be used from main
+# Expose any methods meant to be used from main.
 sync.heavy_computation = lambda: 6 * 7
 ```
 
-Save these files in a *tmp* folder and use `npx mini-coi ./tmp` to reach out that `index.html` and see the following outcome in *devtools*:
+Save these files in a `tmp` folder, ensure [your headers](#http-headers) (just
+use `npx mini-coi ./tmp` and update `index.html`) then see the following
```

Comment:
*and update ...* is not needed ... nothing to update there, just reach localhost to see it?

Reply:
Ah.... coolio. Lemme fix. :wink: